__tests__/pages/ab-testing/demographic-data-survey.test.js

```diff
@@ -76,20 +141,8 @@ describe("Questionnaire démographique", () => {
         )
       ).toBeFalsy()
     })
-    test("une partie des questions seulement est complété => false", () => {
-      const genderValues = [
-        {
-          id: "femme",
-          text: "Femme",
-          isChecked: true,
-        },
-        {
-          id: "homme",
-          text: "Homme",
-          isChecked: false,
-        },
-      ]
 
+    test("une partie des questions seulement est complété => false", () => {
```

Comment:
je l'avais pas vu le petit garnement ! 
`should return false when form is only partially completed`